docs/CHANGELOG.md

```diff
@@ -15,10 +15,10 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 
 ## PLACEHOLDER FOR CELL TYPING RELEASE DATE
 
-* Cell type annotations are now included in each download. 
+* Cell type annotations are now included in each download.
 Cells were annotated using both [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html) and [`CellAssign`](https://github.com/Irrationone/cellassign).
-  * You can find more information about how cell types were annotated in the {ref}`cell type annotation procedures section on the Processing Information page<processing_information:cell type annotation>`. 
-  For more information on locating cell type annotations and any associated processing information in the downloaded objects see {ref}`the Single-cell gene expression file contents page<processing_information:components of a singlecellexperiment object>`.
+  * You can find more information about how cell types were annotated in the {ref}`cell type annotation procedures section on the Processing Information page<processing_information:cell type annotation>`.
+  For more information on locating cell type annotations and any associated processing information in the downloaded objects see {ref}`the Single-cell gene expression file contents page<sce_file_contents:components of a singlecellexperiment object>`.
```

Comment:
Just noting that these changes should go into the cell type release docs

Reply:
Yup! Already there - https://github.com/AlexsLemonade/scpca-docs/pull/223/commits/736849348b20a43f8cbc1403f401e9f5a97cf087